examples/04.MQTT/mqtt.cc

```diff
@@ -114,6 +116,13 @@ void __cheri_compartment("mqtt_example") example()
 	auto heapAtStart =
 	  heap_quota_remaining(STATIC_SEALED_VALUE(mqttTestMalloc));
 
+	Debug::log("Generating client ID...");
+	// Prefix with something recognizable, for convenience.
+	memcpy(clientID, clientIDPrefix, clientIDPrefixLength);
```

Comment:
Please don't use `memcpy`, use types that know their size and make this correct by construction.

Reply:
Any suggestion to avoid the `memcpy`?